chains/celo/listener_test.go

```diff
@@ -30,3 +90,73 @@ func TestListener_start_stop(t *testing.T) {
 	// Initiate shutdown
 	l.close()
 }
+
+// Testing transaction Block hash
+func TestListener_TransactionBlockHash(t *testing.T) {
+
+	l := createTestListener(t)
+	err := l.start()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Create and submit a new transaction and return the signed transaction hash
+	hash := newTransaction(t, l)
+
+	// Get the transaction
+	tx, _, err := l.conn.Client().TransactionByHash(context.Background(), hash)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
I don't think you need this, same as above, you don't need to fetch the tx to fetch the recipt as long as you have the tx hash already